### ISSUE TITLE
Draggable property is ignored on custom event renderer

### ISSUE THREAD

**Issue by vineettadakamalla** [aldabil21/react-scheduler#296]
I added a custom event render and observed that events are draggable even if I set the individual events draggable to be false. 
How do I disable dragging at an event level using custom renderer? 

{
    event_id: 1131,
    title: "Unavailable",
    editable: false,
        draggable: false,
        deletable: false,
    start: new Date(new Date(new Date().setHours(15)).setMinutes(30)),
    end: new Date(new Date(new Date().setHours(17)).setMinutes(30)),
    admin_id: 1,
  }
----
 eventRenderer={({ event, ...props }) => {  
            return   <div
            style={{
              display: "flex",
              flexDirection: "column",
              justifyContent: "center",
              height: "100%",
              background: "#0f452291",
              margin: '2px'
            }}
            {...props}
            onClick={() =>{
              console.log(event);
            }}
          >
             <div>{event.title}</div>
             </div>
        }}

**Comment by uxbert-fe** [comment]
Hi
According to the code above, you are spreading the `props` from `eventRenderer`, which enables drag by default.
If you want an event to be not draggable, you can control this on event level or calendar level, check readme this section

![image](https://github.com/aldabil21/react-scheduler/assets/115465591/02cbd296-8e65-42f3-810a-a31e090e394a)


Feel free to re-open is you still has issue

**Comment by vineettadakamalla** [comment]
Hi @aldabil21 - Thanks for the response. I'm setting draggable as false at my event level. I also tried setting props.draggable == false in the event renderer. I even tried setting at the calendar level. None of them are working.

If I remove the custom event renderer, it works. 

My sample event
{
event_id: 1131,
title: "Unavailable",
editable: false,
draggable: false,
deletable: false,
start: new Date(new Date(new Date().setHours(15)).setMinutes(30)),
end: new Date(new Date(new Date().setHours(17)).setMinutes(30)),
admin_id: 1,
}

Tried setting props.draggable = false;
eventRenderer={({ event, ...props }) => {
props.draggabe = false; 
return <div
style={{
display: "flex",
flexDirection: "column",
